src/domain/bitcoin/onchain/index.ts

```diff
@@ -23,3 +31,39 @@ export const IncomingOnChainTransaction = ({
       return a
     }, []),
 })
+
+export const IncomingOnChainTransaction = ({
+  confirmations,
+  rawTx,
+  fee,
+  createdAt,
+}: {
+  confirmations: number
```

Comment:
could be its own type also. don't think that's critical / could be done in another PR.